### ISSUE TITLE
Unable to build project using gretty version 2.3.1 

### ISSUE THREAD

**Issue by nileshkhaire** [gretty-gradle-plugin/gretty#269]
I am not able to build project using gretty version 2.3.1 . 

![image](https://user-images.githubusercontent.com/1253716/194095950-44c1a3fd-229e-4d34-825f-4f6a22bb6c79.png)

I searched for this library on the maven central and it seems that version 2.3.1 is not exist. Does it mean version deleted from Maven central ?  As per my knowledge this is not a good practice. 

![image](https://user-images.githubusercontent.com/1253716/194096705-67fc3585-e273-43b5-b3e4-aa70281f55ff.png)

Unfortunately, I don't want to upgrade to latest version or newer version for some reasons. Is there possibility to use version 2.3.1 

Thank you .

**Comment by f4lco** [comment]
Hello @nileshkhaire!

> I searched for this library on the maven central and it seems that version 2.3.1 is not exist. Does it mean version deleted from Maven central ? As per my knowledge this is not a good practice.

We never published Gretty 2.x to Maven Central, that is why you cannot find it.
Back in the day, we published to JCenter / Bintray repositories.
I'm sorry to break the news to you, but Bintray was sunset in May 2021.
See #192 and the announcement: https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/

It looks like mvnrepository still maintains some of the links, which are dead by now:

<img width="732" alt="image" src="https://user-images.githubusercontent.com/29141485/194163241-c94b4c73-5ec4-4848-bdf1-7659b464971c.png">

https://mvnrepository.com/artifact/org.gretty/gretty-runner-jetty?repo=jcenter

I suppose your best option is to build from source, and perhaps to deploy to your local Maven repository on your machine, or to deploy the artifacts to a (company internal) private repository.

**Comment by nileshkhaire** [comment]
Thank you for quick r4esponse @f4lco . 

I tried to build from source but I see errors .

Command I used: gradle.bat build

```
* What went wrong:
A problem occurred configuring project ':libs:gretty'.
> Failed to notify project evaluation listener.
   > org.gradle.api.publish.maven.internal.publication.MavenPublicationInternal.getPublishableFiles()Lorg/gradle/api/file/FileCollection;
```

Is there any documentation on how to build the project ? Unfortunately, I didn't find it .

Thank you :)

**Comment by nileshkhaire** [comment]
Hello @f4lco ,

I am still getting the same error. This could be because of proxy or restriction to run executables (like .exe or .dll ) .

As there are lot of efforts involved in building from source finally we have decided to upgrade gretty to 3.0.5 . And it is working fine, obviously we had to upgrade to Tomcat 9 . 

So I am closing this ticket . Hope it is okay for you.

Thank you for your quick support and building this awesome plugin.  :)
